crypto/asn1/asn1_test.cc

```diff
@@ -2412,6 +2412,70 @@ TEST(ASN1Test, LargeString) {
 #endif
 }
 
+
+// Wrapper functions are needed to get around Control Flow Integrity Sanitizers.
```

Comment:
Would implementing/using the `D2I_OF` and `I2D_OF_const` macros make these functions unnecessary. I tried this change locally, and the CFI build passed.

Reply:
https://github.com/aws/aws-lc/commit/74a1acd3666c007e6ae15a6090e076bf05fe3fd0

Reply:
I took a closer look and noticed that the definition of `CHECKED_I2D_OF` is misaligned from [OpenSSL's ](https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/include/openssl/asn1.h#L264-L265) (which doesn't take a `const`).
I tried changing ours to align, but it's not applicable.. This will cause issues for consumers looking to depend on the same version from OpenSSL, so I've changed ours to align with theirs. The macro's still very handy, so I've renamed and moved it to the `asn1_test` file.